docs_app/src/app/custom-elements/migration-timeline/data-access/migration-timeline-struckture/migration-item.ts

```diff
@@ -0,0 +1,163 @@
+Following things should be considered:
+- The link should target the tag e.g. '.../blob/6.0.0-tactical-rc.1/src/...'
+- The link should target the exact line of code e.g. '.../src/.ts#L42'
+  - If multiple lines are effected the link should include all of them e.g. '.../src/.ts#L21-L42'
+type: string
+Example:
+https://github.com/ReactiveX/rxjs/blob/6.0.0-tactical-rc.1/src/internal/Scheduler.ts#L20
+
+@field breakingVersion
+description: semver string of breaking version
+type: string
+
+@field breakingSubjectAction
+description: subjectAction of related `BreakingChange` item
+type: string
+Example:
+
+@field deprecationMsgCode
+description: The <HumanReadableShortMessage> part from the code placed next to `@deprecated` section without the link.
+The Pattern for the text in the source code is: <HumanReadableShortMessage> - see <LinkToDeprecationPage>
+type: string
+
+@field reason
+description: Short explanation of why the deprecation got introduced
+type: string
+
+@field implication
+description: This section is an explanation that accompanies the 'before deprecation' and 'after deprecation' snippets.
+It explains the different between the two versions to the user
+in a detailed way to help the user to spot the differences in code.
+type: string
+
+@field exampleBefore
+description: Code example showing the situation before the deprecation.
+Consider following guide lines:
+- No '$' notation
+- One line between imports asn code
+- const source ... => ... source.subscribe(next: n => console.log(n));
+- It has to work in StackBlitz in the right version
+- Code should log a result
+
+type: string
+Pattern:
+  <imports>
+  <code>
+Example:
+  import {empty} from 'rxjs';
+  empty();
+
+@field exampleAfter
+description: Code example showing the situation before the deprecation
+type: string
+Pattern:
+  <imports>
+  <code>
+Example:
+  import {EMPTY} from 'rxjs';
+  EMPTY;
+*/
+export interface Deprecation extends MigrationItemSubjectUIDFields {
+  itemType: MigrationItemTypeDeprecation;
+  sourceLink: string;
+  breakingChangeVersion: string;
+  breakingChangeSubjectAction: string;
+  deprecationMsgCode: string;
+  reason: string;
+  implication: string;
+  // @TODO Ensure this is the way to handle dependencies to other libs in stackblitz
+  exampleBeforeDependencies?: {[lib: string]: string},
+  exampleBefore?: string;
+  // @TODO Ensure this is the way to handle dependencies to other libs in stackblitz
+  exampleAfterDependencies?: {[lib: string]: string},
+  exampleAfter?: string;
+  notes?: string;
+}
+
+/*
+@Interface BreakingChange
+
+@field itemType
+description: Override of `MigrationItemUIDFields` property `itemType` to MigrationTimeTypeBreakingChange
+type: MigrationTimeTypeBreakingChange
+Example:
+- breakingChange
+
+@field deprecationVersion
+description: semver string of release where deprecation got introduced
+type: string
+
+@field deprecationSubjectAction
+description: subjectAction of related `Deprecation` item
+type: string
```

Comment:
those kind of comments make me really struggle understanding the code. It doesn't really add any value and just bloats the code

Reply:
Removed all comments here:
https://github.com/ReactiveX/rxjs/pull/5128/commits/dea448571df5e713ef78b78eda79522f7faa7b65